src/app/page.tsx

```diff
@@ -1,17 +1,20 @@
 'use server';
 
-import { Uploader } from '@/components/server/Uploader';
-import { CreateUserForm } from '@/components/server/user/CreateUserForm';
-import { UpdateUserForm } from '@/components/server/user/UpdateUserForm';
-import { UserCard } from '@/components/server/user/UserCard';
+import Link from 'next/link';
+import { ListCategory } from '../components/server/category/ListCategory';
+import { ListFiles } from '../components/server/file/ListFiles';
 
 export default async function Home() {
 	return (
 		<main className="flex min-h-screen flex-col items-center justify-between p-24">
-			<CreateUserForm />
-			<UserCard />
-			<UpdateUserForm />
-			<Uploader />
+			<ListCategory />
+			<Link href="/uploadFile" className="inline-block bg-black text-white py-2 px-4 rounded-lg">
```

Comment:
wrapper okolo Linku pls.. všude to nějak styluješ... idealně varianty a přes generiku předat propsy z next/Link